recipes/robotraconteur_companion/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "robotraconteur_companion" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotraconteur/robotraconteur_companion/archive/refs/tags/v0.3.1.tar.gz
+    sha256: cfcf199d1187b30fdd83b5e61e4113da6d819f61a032cfc4ccd5fa463c3d3ef1
+  - url: https://github.com/robotraconteur/robotraconteur_standard_robdef/archive/refs/tags/group1-v1.tar.gz
+    sha256: aad4f122b1240756564d6f3d16258e3fd07febad9e0ceda489c5e32290e64707
+    folder: robdef
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - git
+    - cmake
+    - ninja  # [win]
+    - make  # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - robotraconteur
+    - libboost-devel
+    - yaml-cpp
+    - eigen
+    - openssl  # [unix]
+  run:
+    - eigen
+    - yaml-cpp
+    - robotraconteur
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/RobotRaconteurCompanion/StdRobDef/StdRobDefAll.h  # [not win]
+    - if exist %PREFIX%\\Library\\include\\RobotRaconteurCompanion\\StdRobDef\\StdRobDefAll.h (exit 0) else (exit 1)  # [win]
+    - test -f ${PREFIX}/lib/libRobotRaconteurCompanion.so  # [linux]
```

Comment:
Use `SHLIB_EXT`: https://github.com/conda-forge/ovito-feedstock/pull/14

Reply:
That example doesn't use `SHLIB_EXT` in meta.yaml

Reply:
Same thing, see e.g. https://github.com/conda-forge/gsl-feedstock/blob/b26086f929cf0ea6a21c5bfd6d1e64ef871e4063/recipe/meta.yaml#L47

Reply:
Done